tests/unit_tests/pyanaconda_tests/modules/network/test_module_network.py

```diff
@@ -901,10 +904,14 @@ def test_firewall_config_task_enable(self, execInSysroot):
                                          trusts = [])
             task.run()
 
-            execInSysroot.assert_called_once_with('/usr/bin/firewall-offline-cmd', ['--enabled', '--service=ssh'], root=sysroot)
+            execWithRedirect.assert_called_once_with(
+                '/usr/bin/firewall-offline-cmd',
+                ['--enabled', '--service=ssh'],
+                root=sysroot
+            )
 
-    @patch('pyanaconda.core.util.execInSysroot')
-    def test_firewall_config_task_enable_with_options(self, execInSysroot):
+    @patch('pyanaconda.modules.network.firewall.installation.execWithRedirect')
+    def test_firewall_config_task_enable_with_options(self, execWithRedirect):
```

Comment:
I recommend to rename the test arguments to something like `exec_mock`, so we don't have to rename them again later.